Rakefile

```diff
@@ -4,6 +4,7 @@ require "rake/testtask"
 Rake::TestTask.new do |t|
   t.libs.push 'spec'
   t.test_files = FileList['spec/**/*_spec.rb']
+  t.warning = false
```

Comment:
Any particular reason to turn of warnings here?

Reply:
No particular reason, I can revert the change if you want?

Reply:
I prefer keeping warnings on, but I'll let @maartenvanvliet have the final say.

Reply:
I've reverted the change and merged newest master so tests are run

Reply:
Thx, yes I prefer keeping the warnings

Reply:
done